lighthouse-core/audits/metrics/total-blocking-time.js

```diff
@@ -6,24 +6,28 @@
 'use strict';
 
 const Audit = require('../audit.js');
-const CumulativeLQD = require('../../computed/metrics/cumulative-long-queuing-delay.js');
+const ComputedTBT = require('../../computed/metrics/total-blocking-time.js');
+const i18n = require('../../lib/i18n/i18n.js');
 
-// TODO(deepanjanroy): i18n strings once metric is final.
-const UIStringsNotExported = {
-  title: 'Cumulative Long Queuing Delay',
-  description: '[Experimental metric] Total time period between FCP and Time to Interactive ' +
-      'during which queuing time for any input event would be higher than 50ms.',
+const UIStrings = {
+  /** The name of the metric that calculates the total duration of blocking time for a page. Blocking times are time periods when the page would be slow at responding to user input (clicks, taps, and keypresses will feel janky). Shown to users as the label for the numeric metric value. Ideally fits within a ~40 character limit. */
+  title: 'Total Blocking Time',
```

Comment:
@exterkamp did we decide we aren't translating metrics or just abbreviations? what's the deal now?

Reply:
friendly ping on this one, from @brendankenny's request for better translator explanation I'm guessing we *do* want them translated afterall, so maybe it was just abbreviations I'm remembering?

Reply:
ha, I did not consult @exterkamp when suggesting the description change. I think we might be punting on metrics/abbreviations until after placeholders (which will itself be after this PR)

Reply:
Should I be removing i18n code and comments for the title then, or is it fine to leave it as it is?

Reply:
> Should I be removing i18n code and comments for the title then, or is it fine to leave it as it is?

no, I think should leave as it is here and we'll go through them in the future when we deal with all of them at once (after #9114 some time)